packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -205,6 +208,37 @@ export default class AgoraIpOrderCtrl {
     this.trackStep(3);
   }
 
+  getServiceRegion() {
+    this.model.selectedServiceRegion = null;
+    let request = null;
+    const { serviceName } = this.model.selectedService;
+
+    switch (this.model.selectedService?.type) {
+      case PRODUCT_TYPES.dedicatedServer.typeName:
+        request = this.IpAgoraOrder.getDedicatedServerRegion(serviceName);
+        break;
+      case PRODUCT_TYPES.vps.typeName:
+        request = this.IpAgoraOrder.getVpsServerRegion(serviceName);
+        break;
+      case PRODUCT_TYPES.privateCloud.typeName:
+        request = this.IpAgoraOrder.getDedicatedCloudServerRegion(serviceName);
+        break;
+      default:
+        request = this.$q.when(null);
+        break;
+    }
+
+    request.then((region) => {
+      this.model.selectedServiceRegion =
+        this.model.selectedService?.type ===
+        PRODUCT_TYPES.dedicatedServer.typeName
+          ? this.$translate.instant(`ip_region_${region}`)
+          : this.ovhManagerRegionService.getTranslatedMicroRegionLocation(
+              region,
+            );
```

Comment:
What's the behavior is region is `null` (`default` switch case), `ovhManagerRegionService.getTranslatedMicroRegionLocation` returns null or `''` ?

Reply:
It will return null as in getTranslatedMicroRegionLocation the return is like this : 

` return translatedMicroRegionLocation ||  region;`

so nothing will be displayed